src/nnet3/nnet-simple-component.h

```diff
@@ -928,7 +928,7 @@ class LinearComponent: public UpdatableComponent {
  public:
   virtual int32 InputDim() const { return params_.NumCols(); }
   virtual int32 OutputDim() const { return params_.NumRows(); }
-
+  const CuMatrix<BaseFloat> &LinearParams() const { return params_; }
```

Comment:
Can't you just use Params()?

Reply:
I am not much sure of editing the function name.